docs/atomic.md

```diff
@@ -0,0 +1,6 @@
+# Atomic updates
+By design git-sync uses symlinks to atomically switch git content between versions.
```

Comment:
... This means that your application will always have a consistent view of a single git revision.  When a new revision is available, the entire contents are pulled to a staging area and then the symlink is changed, at which point your application will see only the new revision.